src-browser/os.js

```diff
@@ -1,5 +1,26 @@
 import os from 'os-browserify';
+import { _uptime } from './_process_common.js';
+// https://github.com/nodejs/node/blob/14aed60941da978346de286b3d5b814327944d38/lib/internal/buffer.js#L40
+const float32Array = new Float32Array(1);
+const uint8Float32Array = new Uint8Array(float32Array.buffer);
+float32Array[0] = -1;
+var _endianness = uint8Float32Array[3] === 0 ? 'BE' : 'LE';
```

Comment:
Similar concept, slightly similar implementation:

```suggestion
const _endianness = new Uint8Array(new Uint16Array([1]).buffer)[0] === 1 ? 'LE' : 'BE';
```